### DISCUSSION TITLE
USART Tx not working on F446RE Nucleo 64

### DISCUSSION THREAD

**Discussion by matutef1** [grblHAL/STM32F4xx#195] [Q&A] [answered]
Hello, this is my first grblHAL attempt. I'm building a F446RE Nucleo 64 + Protoneer v3 with pretty stock configuration.

I enabled the `#define BOARD_PROTONEER_3XX` in _my_machine.h_, disabled `#define USB_SERIAL_CDC` just in case and set `#define SPINDLE0_ENABLE      SPINDLE_NONE` since I saw it was using the UART Tx pin, so I wanted to discard that issue.

Firmware seems to be running good, I put some breakpoints on `UART0_IRQHandler` and they're hit as expected, received data is ok, but nothing is transmitted back.

Any help will be appreciated.
Thanks!

**Comment by terjeio** [comment, marked as answer]
Comment out AUXOUTPUT0 port and pin
